packages/react/dropdown-menu/src/DropdownMenu.stories.tsx

```diff
@@ -668,43 +668,67 @@ export const MultipleItemsAsDialogTriggers = () => {
 };
 
 export const CheckboxItems = () => {
-  const checkboxItems = [
-    { label: 'Bold', state: React.useState(false) },
-    { label: 'Italic', state: React.useState(true) },
-    { label: 'Underline', state: React.useState(false) },
-    { label: 'Strikethrough', state: React.useState(false), disabled: true },
+  const corvids = [
+    { label: 'Crows' },
+    { label: 'Ravens' },
+    { label: 'Magpies' },
+    { label: 'Jackdaws' },
```

Comment:
nit: If we're only going to deal with the labels now, let's just make it an array of strings instead.